conans/test/integration/toolchains/intel/test_using_cmake.py

```diff
@@ -0,0 +1,22 @@
+from conans.client.toolchain.cmake.base import CMakeToolchainBase
+
+from ._base import BaseIntelTestCase
+from ..test_content import conanfile_py, create_library
+
+
+class CMakeIntelTestCase(BaseIntelTestCase):
+
+    def test_use_cmake_toolchain(self):
+        create_library(self.t)
+
+        self.t.save({'conanfile.py': conanfile_py.format(generator='None'),
+                     'profile': self.profile})
+
+        # Build in the cache
+        self.t.run('create . foobar/version@ --profile:host=profile')
+
+        # Build locally
+        self.t.run('install . foobar/version@ --profile:host=profile')
+        self.t.run_command('cmake . -G "Visual Studio 15 2017" '
+                           '-DCMAKE_TOOLCHAIN_FILE={}'.format(CMakeToolchainBase.filename))
+        self.t.run_command('cmake --build .')
```

Comment:
Can we add some checks about the compiler itself, some compiler_id preprocessor or something, that shows the binary was really compiled with icc?

Reply:
I changed to follow the same pattern - now check for `__INTEL_COMPILER` macro in the executable's output